### ISSUE TITLE
[Bug] Puppeteer downloading

### ISSUE THREAD

**Issue by b5414** [berstend/puppeteer-extra#672]
**Describe the bug**

The browser downloads each installation with such dependencies (attached below); even if I don't want to load `puppeteer-extra`...

It's ~500 mb, which are built in lambda every time, which adds some complexity

**Code Snippet**

```javascript
> npm i
// with:
"dependencies": {
	"@sparticuz/chrome-aws-lambda": "^14.4.1",
	"playwright-core": "^1.24.0-alpha-jul-6-2022",
	"puppeteer-extra-plugin-stealth": "^2.10.2"
},
```

**Versions**

```js
> npm list
+-- @sparticuz/chrome-aws-lambda@14.4.1
+-- playwright-core@1.24.0-alpha-jul-6-2022
+-- puppeteer-extra-plugin-stealth@2.10.2
```

**Comment by berstend** [comment]
ah, `puppeteer-extra-plugin-stealth` has `puppeteer-extra-plugin` as a dependency which lists `puppeteer-extra` as a peer dependency which lists `puppeteer` as a peer dependency.

Seems like peer dependencies are automatically installed, that behavior changed at some point if I recall correctly.

Will remove those peer dependencies or mark them as optional.

Quick fix:
set `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true`

**Comment by berstend** [comment]
Closing in favor of #527

**Comment by berstend** [comment]
Fixed in `playwright-extra@4.3.2`

**Comment by b5414** [comment]
Very nice, thank you 😘
